extensions/Worker.Extensions.ServiceBus/src/Worker.Extensions.ServiceBus.csproj

```
@@ -6,16 +6,26 @@
     <Description>Azure Service Bus extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>5.7.0</VersionPrefix>
+    <VersionPrefix>5.9.0</VersionPrefix>
 
     <!--Temporarily opting out of documentation. Pending documentation-->
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
   </PropertyGroup>
 
   <Import Project="..\..\..\build\Extensions.props" />
-  
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.13.0-alpha.20230203.1" />
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.6.0" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+    <ProjectReference Include="..\..\..\src\DotNetWorker.Core\DotNetWorker.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <SharedReference Include="../../Worker.Extensions.Shared/Worker.Extensions.Shared.csproj" />
```

I think you might have copied this from the blob PR and I made a mistake, should be consistent with the rest:
```suggestion
    <SharedReference Include="..\..\Worker.Extensions.Shared\Worker.Extensions.Shared.csproj" />
```